GOVERNANCE.md

```diff
@@ -2,7 +2,7 @@
 
 This document outlines a set of policies in order to provide a level playing field and open process for contributors to join the OpenFGA project.
 
-Governance activities are currently performed manually. In the future we might use [terraform](https://www.terraform.io/) or similar technology to automate tasks, but we're currently a small team and iterating quickly - for now it makes sense to do things in a way that doesn't scale.
+Governance activities are currently performed manually. In the future we might use [Terraform](https://www.terraform.io/) or similar technology to automate tasks, but we're currently a small team and iterating quickly - for now it makes sense to do things in a way that doesn't scale.
```

Comment:
How would we use terraform for this?

Reply:
We might want to use: https://registry.terraform.io/providers/integrations/github/latest/docs to manage our repos, the teams, members and access - that would help make access changes more transparent and auditable, as it would mean for a contributor to be a maintainer, that would be a PR that would be approved by one (or 2 or more) admins.